shared/chat/conversation/attachment-get-titles/index.js

```diff
@@ -194,4 +180,4 @@ const stylesInputMobile = {
   paddingRight: 20,
 }
 
-export default GetTitles
+export default Kb.HeaderOrPopup(GetTitles)
```

Comment:
on master looks pretty crappy, its a popup so it looks weirdly squished, instead just make this a regular route